### PR TITLE
Update doc to mark dist upgrade, java update, and migration of home dirs as done

### PR DESCRIPTION
**Pull request by gruunday**

### CODE REVIEW

**Comment by VoyTechnology** [comment]
When merging can a better name be provided for the merge commit? Would be pretty hard to figure out what happened from the current name

**Comment by gruunday** [comment]
@VoyTechnology  Yes apologizes

**Comment by gruunday** [comment]
Not urgent. I am sure someone will double check in the morning

**Review by butlerx — CHANGES_REQUESTED** [review]
Php was also done

File: docs/plans/TODO-2017.md [hunk 1]
```diff
@@ -28,7 +28,8 @@
     - [ ] ldap
   - [ ]  Docker services
     - [ ] hybridIRC
-    - [ ] bitlbee
+    - [X] bitlbee
```
Comment (butlerx) [inline docs/plans/TODO-2017.md]: this is installed but user config needs to be moved and dns changed

Comment (gruunday) [reply docs/plans/TODO-2017.md]: I installed bitlbee last night.... was it already there? I installed a docker one with the facebook bitlbee too. Currently works

Comment (butlerx) [reply docs/plans/TODO-2017.md]: yes `/etc/docker-compose/services/bitlebee` it also worked with the fb plugin installed

**Review by butlerx — APPROVED** [review]

**Pull request merged**